reference/api/overview.md

```diff
@@ -1,45 +1,115 @@
 # API reference
 
-Welcome to the Meilisearch API documentation.
+Welcome to the Meilisearch API documentation. If you are new to Meilisearch, check out our [quick start guide!](/learn/getting_started/quick_start.md)
 
-::: tip
+Meilisearch is a RESTful API. This page describes the general behavior of the API.
 
-Check out [the FAQ](/resources/faq.md) for answers to some common questions 💡
+## OpenAPI
 
+The Meilisearch OpenAPI specifications:
+
+- [JSON](https://bump.sh/doc/meilisearch.json)
+- [YAML](https://bump.sh/doc/meilisearch.yaml)
+
+## Document conventions
+
+The API documentation uses the following conventions:
+
+- Curly braces (`{}`) represent path parameters, e.g., GET `/indexes/{index_uid}`
+- Required fields are marked by an asterisk (`*`)
+- Placeholder text is in uppercase characters with underscore delimiters, e.g., `MASTER_KEY`
+
+## Authorization
+
+Once you [provide Meilisearch with a master key at launch](/learn/security/master_api_keys.md#protecting-a-meilisearch-instance), you must include the `Authorization` header for all your API requests. The `Authorization` header provides the credentials needed to access a protected resource.
+
+<CodeSamples id="authorization_header_1" />
+
+While the master key grants full access to your Meilisearch instance, it should only be used for managing the [`/keys`](/reference/api/keys.md) route. We recommend using API keys for most of your day-to-day operations. API keys grant access to specific indexes, routes, and endpoints.
+
+::: note
+ v0.24 and below use the `X-MEILI-API-KEY: apiKey` authorization header:
+<CodeSamples id="updating_guide_check_version_old_authorization_header" />
 :::
 
+[To learn more about keys and security, refer to our dedicated guide.](/learn/security/master_api_keys.md)
+
+## Pagination
+
+Meilisearch paginates all GET routes that return multiple resources, e.g., GET `/indexes`, GET `/documents`, GET `/keys`, etc. This allows you to work with manageable chunks of data. All these routes return 20 results per page, but you can configure it using the `limit` query parameter. You can move between pages using `offset`.
+
+All paginated responses contain the following fields:
+
+| Name         | Type    | Description                  |
+| :----------- | :------ | :--------------------------- |
+| **`offset`** | Integer | Number of resources skipped  |
+| **`limit`**  | Integer | Number of resources returned |
+| **`total`**  | Integer | Total number of resources    |
+
+### `/tasks` endpoint
+
+The `/tasks` endpoint uses a different type of pagination; hence the response contains different fields. You can read more about it in the [tasks API reference](/reference/api/tasks.md#get-tasks).
```

Comment:
```suggestion
Since the `/tasks` endpoint uses a different type of pagination, the response contains different fields. You can read more about it in the [tasks API reference](/reference/api/tasks.md#get-tasks).
```